pagination/src/main/java/com/cerner/beadledom/pagination/parameters/PaginationParameters.java

```diff
@@ -0,0 +1,45 @@
+package com.cerner.beadledom.pagination.parameters;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.UriInfo;
+
+/**
+ * Query parameters for an endpoint with offset based pagination.
+ *
+ * <p>This class can be added with the @BeanParam annotation to add the query parameters needed
+ * to support offset based pagination. Allows for injection of default values for limit and offset.
+ * *NOTE* This will not work with configured field names.
```

Comment:
Might be good to have the note in it's own paragraph so it stands out a bit more.